sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.aest.controller.updateProfileInformation.e2e-spec.ts

```diff
@@ -0,0 +1,266 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { IdentityProviders } from "@sims/sims-db";
+import * as faker from "faker";
+
+interface StudentProfileUpdateInfo {
+  givenNames: string;
+  lastName: string;
+  birthdate: string;
+  email: string;
+  noteDescription: string;
+}
+
+describe("StudentAESTController(e2e)-updateProfileInformation", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let studentProfileUpdateInfo: StudentProfileUpdateInfo;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    studentProfileUpdateInfo = {
+      givenNames: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      birthdate: "1990-01-15",
+      email: faker.internet.email(),
+      noteDescription: faker.datatype.uuid(),
+    };
+  });
+
+  it("Should allow the student profile update when the student is a Basic BCeID user.", async () => {
+    // Arrange
+    const student = await saveFakeStudent(db.dataSource);
+    student.user.identityProviderType = IdentityProviders.BCeIDBasic;
+    await db.user.save(student.user);
+    const token = await getAESTToken(AESTGroups.OperationsAdministrators);
+    const endpoint = `/aest/student/${student.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .patch(endpoint)
+      .send(studentProfileUpdateInfo)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+
+    const updatedStudent = await db.student.findOne({
+      select: {
+        id: true,
+        birthDate: true,
+        user: {
+          id: true,
+          firstName: true,
+          lastName: true,
+          email: true,
+          createdAt: true,
+          identityProviderType: true,
+          isActive: true,
+          updatedAt: true,
+          userName: true,
+        },
+        sinValidation: { id: true },
+      },
+      relations: { user: true, sinValidation: true },
+      where: {
+        id: student.id,
+        notes: { description: studentProfileUpdateInfo.noteDescription },
```

Comment:
It's fine to remove "notes" in the "where" statement since "id" is already provided.

Reply:
Just the `id` of the student won't do it @lewischen-aot since a student can have multiple notes associated with it and I am looking for a specific note with the unique `noteDescription` that I provided.